src/utils/csp.js

```diff
@@ -19,6 +19,7 @@ function buildPolicies() {
       "www.googletagmanager.com",
       "www.google-analytics.com",
       "www.gstatic.com",
+      "'sha256-y7pLeNIruC+hCYcWLjrAKfMTQoptl6BVn8PcBOXd+aw='", // GA inline script
```

Comment:
interesting, it needs the inner set of single quotes huh?

Reply:
Right, values without quotes are considered domains, values with quotes (like 'self') are handled separately.